content/end.html

```diff
@@ -12,37 +12,37 @@ <h2> Acknowledgements</h2>
 </p>
 
 <div class="vb"></div>
-
 <p class="full">
-<b>Patrick Lam</b>, director of Software Engineering, for confidentially
-managing the data and unbiasedly advising the project from start to finish.
+<b>Andy Zhang</b> for allowing us to fork the 2018 profile site, saving us time
+and allowing us to focus on making this profile more personal than ever before.
+Also for reviewing our initial drafts and for being available to discuss the
+profile over messenger.
 </p>
 
 <div class="hvb"></div>
 <p class="full">
-<b>Christopher Luc</b> for setting up the processing architecture and advising
-about the best statistical methodologies to apply, and <b>Richard Wong</b> for
-helping write processing scripts.
+<b>Patrick Lam, Derek Rayside</b>, and all other SEs for providing feedback on the
+initial drafts of the class profile.
 </p>
-<div class="hvb"></div>
 
+<div class="hvb"></div>
 <p class="full">
-<b>Joey Loi</b> and <b>Atef Chaudhury</b> for inspiring me to create this class
-profile. Much of the process and structure of this was borrowed from their
-Systems Design Engineering
-2017 Class Profile.
+<b>Kashish Goel</b> for starting discussions and early efforts on this class profile
+back in the fall of 2018. Though you transferred out, we appreciate your
+contributions and wish you didn't leave us!
 </p>
 
 <div class="hvb"></div>
 <p class="full">
-<b>Jenny Sun, Derrek Chow, Kaitlyn Yong</b>, and <b>Bo Peng</b> for providing feedback on the
-initial drafts of the class profile.
+<b>Andrew Xia</b> for sharing resources, ideas, and updates as you worked on the ECE
+2021 Class Profile alongside us. We loved sharing ideas and collaborating with you!
```

Comment:
```suggestion
2021 Class Profile alongside us. We loved exchanging ideas and collaborating with you!
```
(repeat of verb "share"